addons/l10n_in/models/company.py

```diff
@@ -99,8 +99,18 @@ def _inverse_l10n_in_is_gst_registered(self):
                 ]
                 self._activate_l10n_in_taxes(gst_group_refs, company)
                 # Set sale and purchase tax accounts when user registered under GST.
-                company.account_sale_tax_id = self.env['account.chart.template'].with_company(company).ref('sgst_sale_5').id
-                company.account_purchase_tax_id = self.env['account.chart.template'].with_company(company).ref('sgst_purchase_5').id
+                if (
+                    account_sale_tax := self.env["account.chart.template"]
+                    .with_company(company)
+                    .ref("sgst_sale_5", raise_if_not_found=False)
+                ):
+                    company.account_sale_tax_id = account_sale_tax.id
```

Comment:
```suggestion
                account_sale_tax := self.env["account.chart.template"].with_company(company).ref("sgst_sale_5", raise_if_not_found=False)
                if account_sale_tax:
                    company.account_sale_tax_id = account_sale_tax.id
```

Much easier to read like that